doc/policies/admin.rst

```diff
@@ -445,3 +445,23 @@ The usual setup that one administrative account has only this single policy
 and is only used for triggering challenges.
 
 New in version 2.17.
+
+.. _admin_policy_2step:
+
+hotp_2step and totp_2step
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+type: string
+
+This allows or forces the administrator to enroll a smartphone based token in two steps.
+In the second step the smartphone generates a part of the OTP secret, which the administrator
+needs to enter. (see :ref:`2step_enrollment`).
```

Comment:
Maybe we should add a hint that this only works with the privacyIDEA authenticator?

Reply:
This is why I added the complete section 2step_enrollment.

At this point I wanted to explain the policy itself and not the complete 2step process

Reply:
Do you think this is enough or should we refer to the privacyIDEA Authenticator?
We have the reference to the 2step enrollment process, where the privacyIDEA Authenticator is required.